### ISSUE TITLE
Framework to be used for the official website

### ISSUE THREAD

**Issue by kakirastern** [wildfirepy/wildfirepy.org#3]
As confirmed in the community meeting on May 31st, 2020, we have agreed on using Facebook's React (https://reactjs.org/) as the framework for the official `WildfirePy` website to be launched later in the year.

**Comment by ram-nad** [comment]
I am sorry I must have discussed this in the meeting, but our website will be more on the static site, so will React be a good option?

**Comment by kakirastern** [comment]
No worries, should not be a problem, and I was thinking we could do something like this: https://www.gatsbyjs.org/blog/2017-09-18-gatsby-modern-static-generation

**Comment by ram-nad** [comment]
Ok yes, gatsby might be a good option. It didn't occur to me earlier.

**Comment by kakirastern** [comment]
Yeah, it's React-based

**Comment by ram-nad** [comment]
But if we are still open for exploration, [Hugo](https://gohugo.io/) will be good too.

**Comment by kakirastern** [comment]
I used Hugo... It's a bit not as aesthetically pleasing as Gatsby since it is Python-based, but very easy to set up.

**Comment by kakirastern** [comment]
And Gatsby can be used with sphinx with a plugin (https://www.gatsbyjs.org/blog/2019-02-14-behind-the-scenes-q-and-a/#functional-questions-how-do-i-do-x-with-gatsby): 

> Question: Gatsby transformers support Markdown and asciidoc. Possible support for Sphinx reStructuredText? Answer: Certainly! Gatsby is super pluggable, so whatever content you want to bring to Gatsby, just write a plugin! Check out Creating a Source Plugin.

**Comment by ram-nad** [comment]
Okay then. Maybe yes Gatsby is more comfortable to work with.

**Comment by kakirastern** [comment]
Awesome

**Comment by ram-nad** [comment]
> And Gatsby can be used with sphinx with a plugin (https://www.gatsbyjs.org/blog/2019-02-14-behind-the-scenes-q-and-a/#functional-questions-how-do-i-do-x-with-gatsby):
> 
> > Question: Gatsby transformers support Markdown and asciidoc. Possible support for Sphinx reStructuredText? Answer: Certainly! Gatsby is super pluggable, so whatever content you want to bring to Gatsby, just write a plugin! Check out Creating a Source Plugin.

Even better will reduce our workload

**Comment by kakirastern** [comment]
Yeah, I know... Got the idea from my work with Astropy (Learn Astropy)
